src/test/java/com/redhat/red/build/finder/ReportTest.java

```diff
@@ -74,10 +75,12 @@ public void verifyReports() throws IOException {
         HTMLReport htmlReport = new HTMLReport(files, buildList, ConfigDefaults.KOJI_WEB_URL);
         NVRReport nvrReport = new NVRReport(buildList);
         GAVReport gavReport = new GAVReport(buildList);
+        TargetReport targetReport = new TargetReport(buildList);
 
         htmlReport.outputToFile(new File(folder, "builds.html"));
         nvrReport.outputToFile(new File(folder, "nvr.txt"));
         gavReport.outputToFile(new File(folder, "gav.txt"));
+        targetReport.outputToFile(new File(folder, "target.txt"));
 
```

Comment:
How is this verifying the target report is how we want it?

Reply:
It doesn't have a proper test yet, but this at least makes sure that it's not throwing any exceptions.

Reply:
I've updated the test code to check the contents of the target map.